src/arfsdao_anonymous.ts

```diff
@@ -278,4 +281,14 @@ export class ArFSDAOAnonymous extends ArFSDAOType {
 		const entitiesWithPath = children.map((entity) => new ArFSPublicFileOrFolderWithPaths(entity, hierarchy));
 		return entitiesWithPath;
 	}
+
+	async downloadFileData(fileTxId: TransactionID): Promise<Stream> {
```

Comment:
**For the reviewer**
There's no downloadPrivateFileData & downloadPublicFileData because the only difference is the decryption step for private ones, which is done in another layer

Reply:
This code was de-coupled from the CLI layer https://github.com/ardriveapp/ardrive-core-js/pull/83